app/src/main/java/com/codesoom/assignment/application/ProductReadService.java

```diff
@@ -0,0 +1,24 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+
+import java.util.List;
+
+
+/**
+ * 상품을 조회하는 역할만 담당합니다.
+ */
+public interface ProductReadService {
+
+    /** 모든 상품을 반환합니다. */
+    List<Product> findAll();
+
+    /**
+     * 식별자로 상품을 찾아 반환합니다. 만약 상품을 찾지 못하면 예외를 던집니다.
```

Comment:
`@throws`에서 예외를 설명하고 있으니 이건 생략해 줍시다.
```suggestion
     * 식별자로 상품을 찾아 반환합니다. 
```